pkg/controller/master/virtualmachine/vm_controller.go

```diff
@@ -66,6 +66,7 @@ func (h *VMController) createPVCsFromAnnotation(_ string, vm *kubevirtv1.Virtual
 		pvcAnno.Namespace = vm.Namespace
 		if pvc, err = h.pvcCache.Get(vm.Namespace, pvcAnno.Name); apierrors.IsNotFound(err) {
 			if _, err = h.pvcClient.Create(pvcAnno); err != nil {
+				h.vmController.EnqueueAfter(vm.Namespace, vm.Name, 5*time.Second)
```

Comment:
The idea was to reduce noise in the log. See the comment in the PR summary:

> The PVCs creation will be delayed by 5s after failure. This will minimize the number of log entries. This improvement is implemented as a preventative measure.

But since that situation should not happen anymore, we can remove that if it does not meet with approval.

Reply:
If the `pvcClient.Create` continuously fails, the root cause needs to be identified.

When it is predefined and inevitable, delay operation is ok; otherwise, the default reconciler behaviour seems better.

Generally, here `EnqueueAfter` should works on some kinds of identified `err`, not all `err`.

Reply:
OK, i've removed that.